projects/batfish-common-protocol/src/main/java/org/batfish/common/plugin/PluginConsumer.java

```diff
@@ -194,8 +194,16 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes attrs)
   }
 
   private void loadPluginClass(URLClassLoader cl, String className) throws ClassNotFoundException {
-    cl.loadClass(className);
-    Class<?> pluginClass = Class.forName(className, true, cl);
+    Class<?> pluginClass = null;
+    try {
+      cl.loadClass(className);
+      pluginClass = Class.forName(className, true, cl);
+    } catch (ClassNotFoundException | NoClassDefFoundError e) {
+      //Ignoring this exception is a potentially dangerous hack
+      //but I don't quite know yet why some classes cannot be loaded and how to do things cleanly
+      System.err.println("Couldn't load class " + className + ". Skipping.");
+      return;
+    }
```

Comment:
Pay attention here

Reply:
No strong objection, but integrate with the logging framework rather than syserr.